src/main/java/by/zhydzel/RegistrationPage.java

```diff
@@ -0,0 +1,154 @@
+    WebElement NumberOfInstances;
+    @FindBy(xpath = "//*[@ng-model=\"listingCtrl.computeServer.os\"]")
+    WebElement OperatingSystem;
+    @FindBy(xpath = "//md-option[@value=\"free\"]")
+    WebElement OperatingSystemOption;
+    @FindBy(xpath = "//*[@ng-model=\"listingCtrl.computeServer.class\"]")
+    WebElement MachineClass;
+    @FindBy(xpath = "//md-option[@value=\"regular\"]")
+    WebElement MachineClassOption;
+    @FindBy(xpath = "//md-select[@ng-model=\"listingCtrl.computeServer.series\"]")
+    WebElement Series;
+    @FindBy(xpath = "//*[contains(text(),'N1')]//parent::md-option")
+    WebElement SeriesOption;
+    @FindBy(xpath = "//md-select[@ng-model=\"listingCtrl.computeServer.instance\"]")
+    WebElement MachineType;
+    @FindBy(xpath = "//md-option[@value=\"CP-COMPUTEENGINE-VMIMAGE-N1-STANDARD-8\"]")
+    WebElement MachineTypeOption;
+    @FindBy(xpath = "(//md-checkbox[@aria-label=\"Add GPUs\"])[1]")
+    WebElement AddGPUs;
+    @FindBy(xpath = "//md-select[@placeholder=\"Number of GPUs\"]")
+    WebElement NumberOfGPUs;
+    @FindBy(xpath = "//md-option[@id=\"select_option_437\"]")
+    WebElement NumberOfGPUsOption;
+    @FindBy(xpath = "//md-select[@placeholder=\"GPU type\"]")
+    WebElement GPUType;
+    @FindBy(xpath = "//md-option[@value=\"NVIDIA_TESLA_V100\"]")
+    WebElement GPUTypeOption;
+    @FindBy(xpath = "//md-select[@placeholder=\"Local SSD\"]")
+    WebElement LocalSSD;
+    @FindBy(xpath = "//*[contains(text(),'2x375 GB')]//parent::md-option")
+    WebElement LocalSSDOption;
+    @FindBy(xpath = "//md-select[@placeholder=\"Datacenter location\"]")
+    WebElement DataCenterLocation;
+    @FindBy(xpath = "(//*[contains(text(),' Frankfurt (europe-west3)')]//parent::md-option)[3]")
+    WebElement DataCenterLocationOption;
+    @FindBy(xpath = "//md-select[@placeholder=\"Committed usage\"]")
+    WebElement CommittedUsage;
+    @FindBy(xpath = "//div[@class='md-select-menu-container md-active md-clickable']/descendant::md-option[@ng-value='1'][@value='1']")
+    WebElement CommittedUsageOption;
+    @FindBy(xpath = "(//button[@aria-label=\"Add to Estimate\"])[1]")
+    WebElement AddToEstimate;
+    @FindBy(xpath = "//h2/b")
+    WebElement TotalEstimatedCost;
+    @FindBy(xpath = "//button[@aria-label=\"Email Estimate\"]")
+    WebElement ButtonEmailEstimate;
+    @FindBy(xpath = "//*[@id=\"eposta_adres\"]")
+    WebElement CreatedEmailAddress;
+    @FindBy(xpath = "//input[@ng-model=\"emailQuote.user.email\"]")
+    WebElement EmailForLetter;
+    @FindBy(xpath = "//*[@aria-label=\"Send Email\"]")
+    WebElement ButtonSendEmail;
+    @FindBy(xpath = "//*[@class=\"mailler\"]/li[2]/a")
+    WebElement ReceivedMessage;
+    @FindBy(xpath = "//*[@class=\"mailler\"]/li[2]/a")
+    WebElement ReceivedMessage1;
+    @FindBy(xpath = "//*[contains(text(),'Total Estimated Monthly Cost')]/parent::td/following-sibling::*/h3")
+    WebElement EmailBody;
+
+    public String CheckSumInLetter, CheckSumFirst;
```

Comment:
Looks like initiation of there variables should be moved to test and the code with them bellow..
Moreover better naming for better readability could be used, something like this: 
sumValueActual
sumValueExpected

Reply:
Done